corehq/apps/enterprise/api/resources.py

```diff
@@ -369,10 +369,8 @@ class ODataFeedResource(ODataEnterpriseReportResource):
     Currently includes summary rows as well as individual reports
     '''
 
-    domain = fields.CharField(null=True)
-    num_feeds_used = fields.IntegerField(null=True)
-    num_feeds_available = fields.IntegerField(null=True)
-    report_name = fields.CharField(null=True)
+    domain = fields.CharField()
+    report_name = fields.CharField()
     report_rows = fields.IntegerField(null=True)
```

Comment:
I saw `null=True` is removed from `domain` and `report_name`, why `report_rows` still have it? I checked `report_rows` will be CaseExportInstance or FormExportInstance's `self.get_query().count()`, is it possible for this to return `Null` ?